CHANGELOG.adoc

```diff
@@ -151,6 +151,7 @@ A new edge tool allows to create a flow between two ports.
 - https://github.com/eclipse-syson/syson/issues/466[#466] [syson] Handle implicit specializations from standard libraries for Usages/Definitions.
 - https://github.com/eclipse-syson/syson/issues/667[#667] [diagrams] Add "Show/Hide Inherited Members in Diagrams" action in Diagram Panel, allowing to show/hide inherited members from users models in diagrams.
 - https://github.com/eclipse-syson/syson/issues/680[#680] [diagrams] Add "Show/Hide Inherited Members from Standard Libraries in Diagrams" action in Diagram Panel, allowing to show/hide inherited members from standard libraries in diagrams.
+- https://github.com/eclipse-syson/syson/issues/694[#694] [diagrams] Add a new custom node note among possible node style descriptions.
```

Comment:
Will be part of 2024.11.0, need to be changed after the 2024.9.0 release.

Reply:
Done